src/sample.jl

```diff
@@ -18,8 +18,13 @@ function sample_points(sol::SARSOPSolver, tree::SARSOPTree, b_idx::Int, L, U, t,
     V̲, V̄ = tree.V_lower[b_idx], tree.V_upper[b_idx]
     γ = discount(tree)
 
-    V̂ = V̄ #TODO: BAD, binning method
-    if V̂ ≤ V̲ + sol.kappa*ϵ*γ^(-t) || (V̂ ≤ L && V̄ ≤ max(U, V̲ + ϵ*γ^(-t)))
+    if sol.use_binning
+        V̂ = get_bin_value(tree, b_idx)
```

Comment:
These superscript characters on V are very hard to read. I'd recommend using only ASCII characters, but that is up to @WhiffleFish . There might be some wisdom from [Uncle Bob](https://en.wikipedia.org/wiki/Robert_C._Martin) about clear names in Clean Code.

Reply:
I am not tied to that naming convention at all. I was following what was in the code already. If @WhiffleFish wants, I can change them to more standard names.

Reply:
Sometimes we do Vhat, Vhi, Vlo. Though, I'm not too bothered by this unicode---I do see it taken a little too far sometimes by others. :-)

Reply:
yes, the most important thing is to maintain the same style within the project, so if @WhiffleFish was already doing it, we shouldn't change it now unless we change the whole thing.